pkg/runtime/helper.go

```diff
@@ -169,3 +169,13 @@ func (m MultiObjectTyper) IsUnversioned(obj Object) (bool, bool) {
 	}
 	return false, false
 }
+
+// SetZeroValue would set the object of objPtr to zero value of its type.
```

Comment:
Basically, if we make transfer the function into "pkg/conversion/", it's gonna use "runtime.Object".
However, "pkg/runtime" -> "api/unversioned" -> "pkg/conversion".